packages/stable/src/api/documents/documentsApi.ts

```diff
@@ -60,32 +60,18 @@ export class DocumentsAPI extends BaseResourceAPI<Document> {
    * });
    * ```
    */
-  public search = (
-    query: DocumentSearchRequest
-  ): Promise<DocumentSearchResponse> => {
-    return this.searchDocuments<DocumentSearchResponse>(query);
+  public search = (request: DocumentSearchRequest): DocumentSearchResponse => {
+    return this.searchEndpoint(request);
```

Comment:
Currently doesn't work as expected, any feedback on how I can go about resolving this?

Reply:
reverted